### DISCUSSION TITLE
Mouse support to scroll through applications

### DISCUSSION THREAD

**Discussion by Riscum** [material-shell/material-shell#741] [Ideas]
When opening a new application, the screen with a search box, and a grid with app icons appear. 
I think it would be a good addition to have this grid of application icons to be scrollable with the mouse wheel. One of the strong points of material shell is the mouse friendly operation. This would build on that. 

Additionally having this screen somehow customization could be useful in cases where one app adds multiple launchers with similar long names. List instead of grid, for example could be a simple customization to have here. 

Congrats for the great work. 
Cheers

**Comment by PapyElGringo** [comment]
The mouse scroll is a must have other customization require more work though
